docs/guide/deployer.md

```diff
@@ -150,18 +150,18 @@ For opting in to App, `deployer` supports the following methods:
    Check our [examples/htlc-pyteal-ts](https://github.com/scale-it/algo-builder/tree/master/examples/htlc-pyteal-ts) project to explore how to deploy Stateless Smart Contracts(lsig). In the file `scripts/deploy.ts`, you will find:
 
   ```
-  await deployer.fundLsig('htlc.py',
+  await deployer.fundLsigByFile('htlc.py',
```

Comment:
In the first place, let's use "by name" if possible and also add an example about the second way - by file.

Reply:
ok